### PR TITLE
fix: my feed modal mobile was having some width issues and overflow scroll

### PR DESCRIPTION
**Pull request by rebelchris**
### Describe what this PR does
- The mobile my feed modal had bottom spacing for the footer which caused overscroll
- It was also not positioned at full-width on mobile
- Tested on desktop/mobile changes, yet to try on physical device (based on preview URL)

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [x] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [x] MobileL (420px)
- [x] Tablet (656px)
- [x] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-391 #done

### CODE REVIEW

File: packages/shared/src/components/modals/AccountDetailsModal.module.css [hunk 1]
```diff
@@ -8,3 +8,14 @@
...
+& :global(.modal) {
+    padding: 3rem 0 4.0625rem 0;
+
+@screen responsiveModalBreakpoint {
+  max-height: 40rem;
+  padding-top: 0;
+  padding-bottom: 0;
+}
+}
+}
```
Comment (idoshamun) [inline packages/shared/src/components/modals/AccountDetailsModal.module.css:21]: Something looks odd with indentation here

Comment (rebelchris) [reply packages/shared/src/components/modals/AccountDetailsModal.module.css]: Fixed, for some reason modules never reflect 😅

**Comment by idoshamun** [comment]
I manually checked and saw only one issue, the last category is hidden by the fixed button at the bottom

**Comment by rebelchris** [comment]
@idoshamun I can't get rid of the double scroll, it either affects the small screen web users or the mobile users.
The fix is to set min-height to auto-fill, but as mentioned it affects small screen web users (where they won't see anything)

So leaving it for now we can adjust it later.

**Review by idoshamun — APPROVED** [review]

**Pull request merged**